spock-specs/src/test/groovy/org/spockframework/smoke/extension/PendingFeatureIfExtensionSpec.groovy

```diff
@@ -23,6 +23,27 @@ def bar() {
     result.testsSucceededCount == 0
   }
 
+  def "@PendingFeatureIf marks failing feature as skipped if the condition passes and the test fails even if applied twice"() {
+    when:
+    def result = runner.runWithImports("""import spock.lang.PendingFeature
+import spock.lang.PendingFeatureIf
+
+class Foo extends Specification {
+  @PendingFeatureIf({true})
```

Comment:
I guess it is almost impossible to detect at runtime, that these conditions are identical. Otherwise the same objection as with the normal `@PendingFeature` duplication apply here as well

Reply:
I don't think so either.
These are (just retried to be sure) separate `Class` instances at runtime.
So there is no way to see this except from source maybe.